amsd/interfaces.py

```diff
@@ -11,5 +11,23 @@ class Iholder_file(Interface):
 class Iitem_type(Interface):
     """marker"""
 
+class Imaterial(Interface):
+    """marker"""
+
+class Itechnique(Interface):
+    """marker"""
+
+class Isource_type(Interface):
+    """marker"""
+
+class Ikeyword(Interface):
+    """marker"""
+
```

Comment:
Interfaces are only required for "resources" in clld terminology, i.e. things that need a list and details view - not for any controlled vocabulary. So I'm not sure they are required for "technique" and "material", etc.